### PR TITLE
Update pull_request_template.md

### LINKED ISSUE #3050: Dead link to `versioningdocs.md` in PR template
the [Versioning Docs](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/versioningdocs.md) page referenced in the PR template, has disappeared, specifically: `contributing/versioningdocs.md`

https://github.com/dbt-labs/docs.getdbt.com/blob/416ed54347505b6bfe7f53f3bba51e81928a8aba/.github/pull_request_template.md?plain=1#L12

### PR DESCRIPTION
**Pull request by mirnawong1**
fixing dead link in pr template

Closes #3050

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Uncomment if you're publishing docs for a prerelease version of dbt (delete if n
 - [ ] Add versioning components, as described in [Versioning Docs](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/versioningdocs.md)
 - [ ] Add a note to the prerelease version [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/guides/migration/versions)
 -->
-- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
```
Comment (nghi-ly) [inline .github/pull_request_template.md:15]: hey @mirnawong1 : i just tested this ^ link. it works, it's not a "dead link" :

link

am i missing something? ty!

**Review by nghi-ly — COMMENTED** [review]
thanks for looking into this! left one clarifying question

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ Uncomment if you're publishing docs for a prerelease version of dbt (delete if n
 - [ ] Add versioning components, as described in [Versioning Docs](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/versioningdocs.md)
```
Comment (mirnawong1) [inline .github/pull_request_template.md]:
```suggestion
- [ ] Add versioning components, as described in [Versioning Docs](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-entire-pages)
```

File: .github/pull_request_template.md (hunk 1 above)
Comment (mirnawong1) [reply .github/pull_request_template.md:15]: oh right! it's the commented out section 'about versioning' that's dead. updated it !

File: .github/pull_request_template.md (hunk 2 above)
Comment (mirnawong1) [reply .github/pull_request_template.md]: this is the one @nghi-ly !

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -12,7 +12,7 @@ Uncomment if you're publishing docs for a prerelease version of dbt (delete if n
...
 - [ ] Add a note to the prerelease version [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/guides/migration/versions)
 -->
-- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
+- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-entire-pages) so my content adheres to these guidelines.
```
Comment (nghi-ly) [inline .github/pull_request_template.md]: should this update be reverted then?

Comment (mirnawong1) [reply .github/pull_request_template.md]: yes will do in 5 mins, just working on something else but on it soon!

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -9,10 +9,10 @@ To learn more about the writing conventions used in the dbt Labs docs, see the [
...
-- [ ] Add versioning components, as described in [Versioning Docs](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/versioningdocs.md)
+- [ ] Add versioning components, as described in [Versioning Docs](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-entire-pages)
 - [ ] Add a note to the prerelease version [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/guides/migration/versions)
 -->
-- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
+- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-entire-pages) so my content adheres to these guidelines.
```
Comment (mirnawong1) [inline .github/pull_request_template.md]:
```suggestion
- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
```

**Comment by mirnawong1** [comment]
hey @nghi-ly , ok should all be ready to re review

**Review by nghi-ly — APPROVED** [review]

**Pull request merged**